search-app/docs/api.md

```diff
@@ -3140,6 +3137,65 @@ Content-Length: 292
 }
 ```
 
+UMM JSON response:
+
+```
+curl -g -i "%CMR-ENDPOINT%/variables.umm_json?variable_name=Variable1234&pretty=true"
+HTTP/1.1 200 OK
+Content-Type: application/vnd.nasa.cmr.umm_results+json;version=1.9; charset=UTF-8
```

Comment:
Currently, UMM-Var schema versioning by itself is not implemented in CMR, as a result UMM-Var is in the same version as the UMM-C, i.e. at 1.9. This is of course wrong and that's why we have issues to implement UMM-Var versioning. Once that is done, this version will be corrected. This is just what CMR currently returns.